modules/client/src/node.ts

```diff
@@ -19,6 +19,9 @@ import { Logger } from "./lib/logger";
 import { replaceBN } from "./lib/utils";
 import { NodeInitializationParameters } from "./types";
 
+// Include our access token when interacting with these subjects
+const guardedSubjects = ["channel", "transfer"];
```

Comment:
The subjects you care about have to be supplied when you connect to NATS by supplying the signed JWT in the CONNECT request.

```
[94950] 2019/10/03 03:29:53.022012 [INF] Starting nats-server version 2.1.0
[94950] 2019/10/03 03:29:53.022096 [DBG] Go build version go1.12.6
[94950] 2019/10/03 03:29:53.022101 [INF] Git commit [not set]
[94950] 2019/10/03 03:29:53.022357 [INF] Listening for client connections on 0.0.0.0:4220
[94950] 2019/10/03 03:29:53.022365 [INF] Server id is NBSKPK2DWYSQSPSQ3X5QQ4VQXXLJ6EVA272H6HHIOOGRUBLGDECD6PS4
[94950] 2019/10/03 03:29:53.022367 [INF] Server is ready
[94950] 2019/10/03 03:29:53.022376 [DBG] Get non local IPs for "0.0.0.0"
[94950] 2019/10/03 03:29:53.022553 [DBG]  ip=192.168.254.79
[94950] 2019/10/03 03:30:13.338485 [DBG] 127.0.0.1:59432 - cid:1 - Client connection created
[94950] 2019/10/03 03:30:13.355022 [TRC] 127.0.0.1:59432 - cid:1 - <<- [CONNECT {"lang":"typescript","version":"1.2.4","verbose":false,"pedantic":false,"protocol":1,"name":"ts-natsutil-4bfd6be6-1377-470a-b462-23738541b7f7","jwt":"<INSERT SIGNED JWT WITH PERMISSIONS>"}]
```